.eslintrc.js

```
@@ -50,13 +50,16 @@ module.exports = {
+      { extensions: ['.js', '.jsx', '.ts', '.tsx'] },
+    ],
     'react/function-component-definition': 'off',
-    "react/no-array-index-key": "off",
-    "react/jsx-props-no-spreading": "off",
-    "no-use-before-define": "off",
-    "import/no-default-export": "off",
-    "import/prefer-named-exports": "off",
+    'react/no-array-index-key': 'off',
+    'react/jsx-props-no-spreading': 'off',
+    'no-use-before-define': 'off',
+    'import/no-default-export': 'off',
```

ESLint

[error] Duplicate key 'import/no-default-export'.

(no-dupe-keys)